.gitignore

```
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

The environment variables belong only to the environment in which they are installed, .env must be in this list because it is the default file for environment variables.

```suggestion
.env
.env.local
.env.development.local
.env.test.local
.env.production.local
```